unit-test/verify-artifacts.js

```diff
@@ -6,7 +6,7 @@ import { cwd } from '../scripts/script-utils.js'
 const ROOT = join(cwd(import.meta.url), '..')
 const BUILD = join(ROOT, 'build')
 const APPLE_BUILD = join(ROOT, 'Sources/ContentScopeScripts/dist')
-const CSS_OUTPUT_SIZE = 551000
+const CSS_OUTPUT_SIZE = 560000
```

Comment:
Just general question, is there a limit to how much we're willing to increase the CSS output? Or for now this is just to keep track of it?

Reply:
Yea, that's a good question. The original idea was so that we'd know when we hit half-a-megabyte, but since then we've been slowly increasing it as the test fails.